spec/function.dd

```diff
@@ -795,18 +795,22 @@ class Bar : Foo
 ------
 )
 
-        $(P Virtual functions all have a hidden parameter called the
-        $(I this) reference, which refers to the class object for which
-        the function is called.
+        $(P 
+            Non-static (i.e., virtual functions and $(D final) member
```

Comment:
you can use backticks for `$(D final)` now, \`final\`

Reply:
Much nicer, thanks for letting me know. Currently, use of the macro matches the majority of `function.dd`